docs/odissei-tutorial.md

```diff
@@ -0,0 +1,137 @@
+
+# How to use the `sirup` package to manage your IP address in python
+
+When scraping data from the internet, it can be handy to use an IP address that is different from your real IP address. The `sirup` package in python allows you to do this in a flexible way through a VPN service.
+
+**Note**
+When using IP rotationg for webscraping, it is important to respect the law and not overload target website. This means, for instance, moderating the number of requests made to a website in a given time. 
```

Comment:
rotationg -> rotating; "not overload target website" -> "not overload the target website".

do you have some specifics about how many requests per timeframe should be submitted? What are good guidelines? E.g. 30 per minute?